src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -399,7 +399,13 @@ public void close(Collection<TopicPartition> partitions) {
   }
 
   @Override
-  public void setIsStoppedToTrue() {}
+  public void stop() {
+    try {
+      streamingIngestClient.close();
```

Comment:
I think we should use
`StreamingClientProvider.getStreamingClientProviderInstance()
        .closeClient(this.connectorConfig, this.streamingIngestClient);`

Reply:
Good point! I don't fully understand how this would work with one client optimization, even if it's only used in `start()`, I thought it's possible that the client is used by channels from other tasks, then why we close the client during `start() `or `close()`, + @sfc-gh-rcheng as FYI